### PR TITLE
Exclude selectors

### PR DESCRIPTION
**Pull request by sigorilla**
@sergets

### CODE REVIEW

File: command.js [hunk 1]
```diff
@@ -44,6 +50,48 @@ module.exports = function(testBasePath, referencePath, diffPath) {
...
+                                        return {
+                                            x: locations[i].x,
+                                            y: locations[i].y,
+                                            width: size.width,
+                                            height: size.height,
+                                            color: excludeNode.color || '#ff0000'
+                                        };
+                                    })
+                                    .filter(function (rect) {
+                                        return rect.x >0 && (rect.x + size.width) < screenshotSize.width &&
```
Comment (sergets) [inline command.js]: cs :)

File: command.js [hunk 2]
```diff
@@ -44,6 +50,48 @@ module.exports = function(testBasePath, referencePath, diffPath) {
...
+                                    })
+                                    .map(function (rect) {
+                                        return screenshot.fill(rect.x, rect.y, rect.width, rect.height, rect.color);
+                                    });
+                            });
+                        });
+                }, this);
+
+                return vow.all(excludePromises)
+                    .then(function () {
```
Comment (sergets) [inline command.js]: cs, we don't add spaces after `function`

Comment (sigorilla) [reply command.js]: What about add eslint to the project?)

**Comment by sigorilla** [comment]
up

File: command.js [hunk 3]
```diff
@@ -8,18 +8,24 @@ var path = require('path'),
...
-    return function async(x, y, width, height, screenshotId) {
+    return function async(x, y, width, height, screenshotId, options) {
         var selector,
-            screenshot;
+            screenshot,
+            excludeSelectors,
+            argsLength = arguments.length;
 
-        if(arguments.length === 1) {
+        if(argsLength === 1 || (argsLength === 2 && isObject(y))) {
```
Comment (sergets) [inline command.js]: What about:

```js
var args = [].slice.apply(arguments),
    options = args.pop(),
    screenshotId = typeof options === 'string'? options : args.pop(),
    selector = (args.length === 1) && x;

typeof options === 'string' && (options = {});
```

Comment (sigorilla) [reply command.js]: Yeap, it's better.

**Comment by sergets** [comment]
:shipit:

**Comment by sigorilla** [comment]
🚀

**Pull request merged**